docs/man/man1/.files_generated

```diff
@@ -1,31 +0,0 @@
-kubectl-annotate.1
```

Comment:
It is/was used the same as .generated_docs after this PR. It was a way to know which files in the existing repo need to be removed before new docs are put in. If you removed the kubectl-annotate subcommand (from kubectl) and just put new docs in place it would leave the old kubectl-annotate files in the repo.